grind/bin/grind

```diff
@@ -355,6 +355,10 @@ class TestRunner:
                             type=int,
                             default=default_timeout,
                             help="Per-task timeout. The default timeout is %s seconds." % default_timeout)
+        parser.add_argument('--java-version',
+                            type=int,
+                            default=7,
```

Comment:
We can keep the flag as an override for pconfig, sure.

I haven't tried configglue, though I made an effort to not need any extra dependencies for the grind client besides the stdlib.